.idea/modules.xml

```diff
@@ -2,8 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/gwtp-idea-plugin.iml" filepath="$PROJECT_DIR$/gwtp-idea-plugin.iml" />
+      <module fileurl="file://$PROJECT_DIR$/gwtp-idea.iml" filepath="$PROJECT_DIR$/gwtp-idea.iml" />
     </modules>
   </component>
-</project>
-
+</project>
```

Comment:
I think I removed it with git, but it doesn't seem to show in the PR. I'll see what happens when I merge, then I'll make sure it's gone